test/models/report_test.rb

```diff
@@ -3,7 +3,23 @@
 require 'test_helper'
 
 class ReportTest < ActiveSupport::TestCase
-  # test "the truth" do
-  #   assert true
-  # end
+  def setup
+    @user = FactoryBot.create(:user)
+  end
+
+  test 'editable? returns true' do
+    report = FactoryBot.create(:report, user_id: @user.id)
+    assert report.editable?(@user)
+  end
+
+  test 'editable? returns false' do
+    other_user = FactoryBot.create(:other_user)
```

Comment:
ファクトリを別に用意しなくてもこれで別のユーザーを表現できます。

```suggestion
    other_user = FactoryBot.create(:user)
```

ただし、emailが重複すると思うので、sequenceを使うようにしてください。

Reply:
削除しました！